spyder/utils/tests/test_conda.py

```diff
@@ -65,8 +65,17 @@ def test_get_conda_root_prefix():
 
 @pytest.mark.skipif(not running_in_ci(), reason="Only meant for CIs")
 def test_find_conda():
```

Comment:
I think this should be

```python
def test_find_conda():
    pyexec = get_conda_test_env()[1]

    # Remove env vars

    assert find_conda(pyexec=pyexec)

    # Restore env vars
```

for the non-standard conda test to be really sure that `find_conda` is doing its job as expected.

That's because that test env is created with an external conda that we need to manually set in CIs with `CONDA_EXE`.

Reply:
Okay, take a look at my changes.